ibis/expr/operations/relations.py

```diff
@@ -171,7 +172,7 @@ def _check_integrity(values, allowed_parents):
 @public
 class Project(Relation):
     parent: Relation
-    values: FrozenDict[str, Unaliased[Value]]
+    values: FrozenDict[str, NonSortKey[Unaliased[Value]]]
```

Comment:
There may be other places we want to add this restriction, but I didn't want to bother with them in this PR for now.